sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -484,6 +485,20 @@ object DataSourceStrategy {
     }
   }
 
+  private[sql] def getPushedDownFilters(
```

Comment:
You can remove "private[sql]" since we're already in the private package `execution`.